kidviz/models.py

```diff
@@ -260,7 +263,7 @@ def get_observations(cls, course_id, date_from, date_to, tags):
         if tags:
             observations = observations.filter(tags__in=tags)
 
-        return observations
+        return (observations, star_chart_4_obs)
```

Comment:
`observations` is always equal to `star_chart_4_obs`, am I right? If so, why do you return tuple of two duplicated elements?